src/orion/core/io/experiment_branch_builder.py

```diff
@@ -38,7 +38,13 @@ def __init__(self, conflicts, branching_configuration):
         Initialize the ExperimentBranchBuilder by populating a list of the conflicts inside
         the two configurations.
         """
-        self.auto_resolution = branching_configuration.pop('auto_resolution', False)
+        self.auto_resolution = branching_configuration.get('auto_resolution', None)
+
+        if self.auto_resolution is not None:
+            log.info("--auto-resolution is deprecated and will be removed in v0.2.0 of Orion.")
```

Comment:
```suggestion
            log.info("auto_resolution is deprecated and will be removed in v0.2.0.")
```
There should be specifics to cmdline interface in experiment branch builder.

Reply:
You mean shouldn't right?

Reply:
shouldn't yes! :sweat_smile: